Userland/DevTools/HackStudio/Dialogs/Commit/GitCommitDialog.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "GitCommitDialog.h"
+#include <DevTools/HackStudio/Dialogs/Commit/GitCommitDialogGML.h>
```

Comment:
Please use a relative "" import if possible.

Reply:
Since it's auto-generated, CLion doesn't like it being imported like that
![image](https://user-images.githubusercontent.com/71222289/126876693-48f7570d-44fc-4a01-9d7b-b17256b15afc.png)